docs/available_commands.md

```diff
@@ -146,7 +150,32 @@ When passing the targets, names and values in a file, the file must conform to t
                         }
                     }
                 ]
-            }
+            },
+        },
+    ]
+
+Example for the use of version ranges:
```

Comment:
Please also add the representation of this component in the SBOM to this example, so people can understand when version_range applies, you can also add one "negative" and "positive" example,  so it is clear when the version range would apply, respectively not.

Reply:
added two minimal examples